docs/src/pages/components/autocomplete/autocomplete.md

```diff
@@ -101,6 +101,14 @@ Head to the [customization](#customization) section for an example with the `Aut
 
 {{"demo": "pages/components/autocomplete/Asynchronous.js"}}
 
+If the logic is fetching new options at each keystroke,
+using the current value of the textbox to filter on the server,
+you need to disable the built-in filtering of the autocomplete component:
+
+```
```

Comment:
```suggestion
```jsx
```

Reply:
Why do you need to indicate this? Doesn't the documentation show up in the same way? i.e. as formatted code

Reply:
We try to explicit the format anytime possible. I recall trying to add a warning in case it's forgotten. I don't know if it's still in place.

Reply:
Good to know. Thank you for the explanation!